classes/Form/ConfigurationForm.php

```diff
@@ -176,7 +176,7 @@ public function generate()
         $helper->fields_value['GA_USERID_ENABLED'] = Configuration::get('GA_USERID_ENABLED');
         $helper->fields_value['GA_CROSSDOMAIN_ENABLED'] = Configuration::get('GA_CROSSDOMAIN_ENABLED');
         $helper->fields_value['GA_ANONYMIZE_ENABLED'] = Configuration::get('GA_ANONYMIZE_ENABLED');
-        $helper->fields_value['GA_TRACK_BACKOFFICE_DISABLED'] = Configuration::get('GA_TRACK_BACKOFFICE_DISABLED');
+        $helper->fields_value['GA_TRACK_BACKOFFICE_ENABLED'] = Configuration::get('GA_TRACK_BACKOFFICE_ENABLED');
```

Comment:
Changing this settings requires an [upgrade script](https://devdocs.prestashop.com/1.7/modules/creation/enabling-auto-update/) so that settings from previous versions (`GA_TRACK_BACKOFFICE_DISABLED`) are cleaned up an carried over to `GA_TRACK_BACKOFFICE_ENABLED` in the new version.

Reply:
Previous version was never released, is it needed in that case?

Reply:
@eternoendless The setting only made it to the dev version, no need to do an upgrade script I guess?

Reply:
Oh in that case it's not needed. I thought it was already released.